exampleCourse/questions/demo/questionParamsCalculation/question.html

```diff
@@ -0,0 +1,7 @@
+<pl-question-panel>
+  <p> Consider two numbers in the range $\left[ {{params.lower_bound}}, {{params.upper_bound}} \right]$,
```

Comment:
I see you are trying to demonstrate adding the question parameters in the question itself, but given these parameters are not relevant to answering the question, this looks a bit confusing. Since this is the example course intended as a demo to instructors, I think it is ok to list these as a meta-comment (something like: "based on questions parameters, $a$ and $b$ are in the range $\left[ {{params.lower_bound}}, {{params.upper_bound}} \right]$.").

Reply:
I'm not sure if you had something specific in mind with "meta-comment", but I did add this as a <p> after the question text.